pkg/file/builder.go

```diff
@@ -938,6 +939,9 @@ func (b *stateBuilder) routes() {
 func (b *stateBuilder) enterprise() {
 	b.rbacRoles()
 	b.vaults()
+	if b.includeLicenses && !b.isKonnect {
```

Comment:
But be worth adding comments and explaining why we don't fetch license when we're running against Konnect.

Reply:
Added comments.